tests/fixtures/fixture_imodc_executable.py

```diff
@@ -0,0 +1,28 @@
+import subprocess
+import sys
+from collections.abc import Callable
+from pathlib import Path
+
+import pytest
+
+from imod_coupler.__main__ import run_coupler
+
+
+@pytest.fixture(scope="session")
+def imodc_executable(imod_coupler_exec_devel: Path) -> Callable[[Path], None]:
```

Comment:
I think this should be called `run_coupler` or something similar

Reply:
I went for run_coupler_function